app/Http/Controllers/PageController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\View\View;
+
+class PageController extends Controller
```

Comment:
I don't agree with the name. There are or will be a lot of pages in your app that will (or should) not be handled by this controller. This makes the name just too generic. A name like `StaticPageController` would do better.